CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_greenhouse_source v0.8.0
+[PR #31](https://github.com/fivetran/dbt_greenhouse_source/pull/31) includes the following updates:
+
+## Breaking Changes
+- Updated `*_id` fields to be cast as strings to ensure compatibility in downstream joins by avoiding potential type mismatches.
+  - Note: most IDs were previously stored as integers.
+
+## Documentation
+- Update README formatting.
+
+## Under the Hood
+- Added an additional integration test to verify functionality when `greenhouse_using_*` variables are disabled.
```

Comment:
Is this just an integration test? It's a run on integration test data so it's also executing the model too. Maybe we could say something like "Added additional CI/CD run and test scripts for when all `greenhouse_using_*` variables are disabled."

Reply:
I don't understand this comment. What exactly is wrong with saying I updated the integration tests?

Reply:
Fair enough.